package.json

```diff
@@ -10,15 +10,22 @@
     "@types/node": "^12.12.35",
     "@types/react": "^16.9.34",
     "@types/react-dom": "^16.9.6",
+    "express": "^4.17.1",
+    "get-port": "^5.1.1",
+    "http": "0.0.1-security",
+    "ioredis": "^4.16.2",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-scripts": "3.4.1",
+    "shortid": "^2.2.15",
+    "socket.io": "^2.3.0",
     "typescript": "^3.7.5"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
```

Comment:
I just replaced it with test:server until we have tests for the client. I thought it would be confusing to have it default to running the server tests since all the other defaults are for the client half